public-api/db-cdr/generate-cdr/generate-fmh-condition-counts.sh

```diff
@@ -0,0 +1,262 @@
+ where questionnaire_response_id=ob.questionnaire_response_id and
+ observation_source_concept_id=43528652 and value_source_concept_id in (43529842, 43528385))
+ group by 4,5,6"
+
+ # Survey question counts by biological sex for fmh condition, family member
+ # To do delete if not used anymore
+ bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
+ "insert into \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.achilles_results\`
+ (id,analysis_id,stratum_1,stratum_2,stratum_5,stratum_6,count_value,source_count_value)
+ select 0 as id, 3320 as analysis_id, '43528698' as stratum_1, cast(fmh.value_source_concept_id as string) as stratum_2, cast(p.gender_concept_id as string) as stratum_5, concat(fmh.observation_source_concept_id,'.',fmh.value_source_concept_id,'.',fmh.value_source_concept_id) as stratum_6,
+ count(distinct ob.person_id) as count_value,count(distinct ob.person_id) as source_count_value
+ from \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.fmh_metadata\` fmh
+ ,\`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.v_full_observation\` ob JOIN UNNEST(SPLIT(fmh_concepts_to_count,',')) as concepts on cast(ob.value_source_concept_id as string)=concepts
+ join \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.v_person\` p on p.person_id=ob.person_id
+ where exists
+ (select * from \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.v_full_observation\`
+ where questionnaire_response_id=ob.questionnaire_response_id and
+ observation_source_concept_id=43528652 and value_source_concept_id in (43529842, 43528385))
+ group by 4,5,6"
+
+# Survey question counts by age decile for fmh condition, family member
+# To do delete if not used anymore
+bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
+"insert into \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.achilles_results\`
+(id,analysis_id,stratum_1,stratum_2,stratum_5,stratum_6,count_value,source_count_value)
+select 0 as id, 3321 as analysis_id, '43528698' as stratum_1, cast(fmh.value_source_concept_id as string) as stratum_2, age_stratum as stratum_5,concat(fmh.observation_source_concept_id,'.',fmh.value_source_concept_id,'.',fmh.value_source_concept_id) as stratum_6,
+COUNT(distinct ob.PERSON_ID) as count_value,0 as source_count_value
+from \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.fmh_metadata\` fmh
+,\`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.v_full_observation\` ob JOIN UNNEST(SPLIT(fmh_concepts_to_count,',')) as concepts on cast(ob.value_source_concept_id as string)=concepts
+join \`${BQ_PROJECT}.${BQ_DATASET}.concept\` c on c.concept_id=cast(concepts as int64)
+join \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.survey_age_stratum\` sa on sa.observation_id=ob.observation_id
+where exists
+(select * from \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.v_full_observation\`
+where questionnaire_response_id=ob.questionnaire_response_id and
+observation_source_concept_id=43528652 and value_source_concept_id in (43529842, 43528385))
+group by 4,5,6"
+
+# Calculate skip count of each condition
```

Comment:
This is new query that calculates skip count of each condition by counting sum of mother condition skip + father condition skip + sibling condition skip + ....